### PR TITLE
[docs] Installation for Windows docs improved

### PR DESCRIPTION
**Pull request by jjfumero**
#### Description

Improve the documentation for the installation of TornadoVM on Windows 10/11

#### Problem description

n/ a.

#### Backend/s tested

Mark the backends affected by this PR.

- [X] OpenCL
- [X] PTX
- [X] SPIRV

#### OS tested

Mark the OS where this PR is tested.

- [ ] Linux
- [ ] OSx
- [X] Windows

#### Did you check on FPGAs?

If it is applicable, check your changes on FPGAs.

- [ ] Yes
- [X] No

#### How to test the new patch?

Link to the documentation.

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -521,15 +521,14 @@ To run individual tests:
 
 .. _installation_windows:
 
-TornadoVM for Windows 10/11 using GraalVM
-================================================
+Installing TornadoVM for Windows 10/11
+=============================
 
-**[DISCLAIMER] Please, notice that, although TornadoVM can run on Windows 10/11 it is still experimental.**
+**[DISCLAIMER] Please, notice that, although TornadoVM can run on Windows 10/11, it is still experimental.**
```
Comment (mairooni) [inline docs/source/installation.rst:527]: note instead of notice

**Review by mairooni — CHANGES_REQUESTED** [review]

File: docs/source/installation.rst [hunk 2]
```diff
@@ -600,70 +573,61 @@ The only thing to note is that the GPU driver you are currently using should be
...
+   setvars.cmd
 
-   python %TORNADO_SDK%\bin\tornado -m tornado.examples/uk.ac.manchester.tornado.examples.compute.MatrixMultiplication2D --params="512"
 
-To run individual tests:
+6. Check the installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code:: bash
+   # explore accelerator reachable from TornadoVM
```
Comment (stratika) [inline docs/source/installation.rst:622]:
```suggestion

   # list the accelerator devices that are reachable from TornadoVM
```

Comment (stratika) [reply docs/source/installation.rst:622]: I added a space line in line 622, because the code snippet (beneath it) is not shown.

**Review by stratika — CHANGES_REQUESTED** [review]

File: docs/source/installation.rst [hunk 3]
```diff
@@ -600,70 +573,61 @@ The only thing to note is that the GPU driver you are currently using should be
...
-
-You should see a list of OpenCL and/or CUDA devices available on your system.
-
-Now try to run a simple test. To run examples with Graal JDK 21, TornadoVM uses modules:
-
-.. code:: bash
+   python -m venv .venv
+   .venv\Scripts\activate.bat
+   .\bin\windowsMicrosoftStudioTools2022.cmd
+   python bin\tornadovm-installer --jdk jdk21 --backend=opencl
```
[the remark this answers is not in the document]
Comment (jjfumero) [reply docs/source/installation.rst:614]: I think in the feature we should only allow to install through the installer script. For now, let's maintain the `nmake` approach as well. I will change this.

**Comment by jjfumero** [comment]
All comments resolved.

**Review by stratika — APPROVED** [review]

**Review by mairooni — APPROVED** [review]

**Pull request merged**